helm/prometheus-meta-operator/templates/alertmanager/servicemonitor.yaml

```diff
@@ -3,19 +3,17 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
-    app.kubernetes.io/instance: alertmanager
-    app.kubernetes.io/managed-by: prometheus-meta-operator
-    app.kubernetes.io/name: alertmanager
+    {{- include "labels.common" . | nindent 4 }}
```

Comment:
This adds the team label here so this is scraped by the agent and not the mc prom. I need to test that this does not screw up with helm ownership